src/components/Trigger/__tests__/Table.test.tsx

```diff
@@ -52,13 +56,17 @@ describe('src/components/Trigger/Table', () => {
                 }
             ] }
             onSort={ onSort }
+            sortBy={ {
+                column: 'system',
+                direction: Direction.ASCENDING
+            } }
         />);
 
         expect(onSort).toHaveBeenCalledTimes(0);
         await userEvent.click(screen.getByText(/system/i, {
             selector: 'button span'
         }));
-        expect(onSort).toHaveBeenCalledWith(1, 'name', Direction.ASCENDING);
+        expect(onSort).toHaveBeenCalledWith(1, 'name', Direction.DESCENDING);
```

Comment:
I had to change this test, because it was crashing due to missing `sortBy` prop. While I was at it I also added testing of the ASCENDING --> DESCENDING transition.